src/views/MaintenancePage.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <main class="maintenance-page">
```

Comment:
@gonzalo-plaza this component is not the main from the page, cause the main is app.vue, maybe is confusing if we add main on the views or components, cause the main represent the main content from the app, and this content is a one part from this,.